packages/conform-dom/package.json

```diff
@@ -18,12 +18,13 @@
 	},
 	"scripts": {
 		"build:js": "rollup -c",
-		"build:ts": "tsc",
+		"build:ts": "tsc --emitDeclarationOnly",
```

Comment:
Why do we need to remove `emitDeclarationOnly` from the tsconfig?

Reply:
when running `tsc` with `--noEmit` and `--emitDeclarationOnly` is set in the tsconfig, `tsc` throws an error saying only one can be set at a time.